README.md

```diff
@@ -199,7 +207,7 @@ To use MACEst for a regression task, the following example can be used:
 ### MACEst with sparse data (see notebooks for more details)
 ```python
 import scipy
-from scipy.sparse import csr_matrix 
+from scipy.sparse import csr_matrix s
```

Comment:
is the s here a typo ?

Reply:
I was waiting for this to be fixed before I approve, are you going to change this now or shall I approve anyway ?

Reply:
yes no problem, looks good